src/Dibi/Translator.php

```diff
@@ -153,8 +153,8 @@ public function translate(array $args): string
 
 		$sql = trim(implode(' ', $sql), ' ');
 
-		if ($this->errors) {
-			throw new Exception('SQL translate error: ' . trim(reset($this->errors), '*'), 0, $sql);
+		if ($this->errors !== []) {
+			throw new Exception('SQL translate error: ' . trim((string) reset($this->errors), '*'), 0, $sql);
```

Comment:
changes behavior, can be null

Reply:
I reverted original condition but kept `(string)` for PhpStan because function `reset()` can return `mixed` value.

Reply:
For nonempty `string[]`? I think it must be always string.